server/circleLib.js

```diff
@@ -259,6 +400,16 @@ class CircleLib {
         } catch (err) {Sentry.captureException(new Error(err));}
     }
 
+    //create initial payment record in mongodb
+    async createPaymentRecord(data, CLIENT, DBNAME, Sentry){
+        console.log('creating payment record' + data);
+        const DB = CLIENT.db(DBNAME);
+        try {
+            const myCollection = await DB.collection('fiatPaymentRecords');
```

Comment:
let's keep the same naming convention for DB tables. Looks like we already started using "snake case" i.e. `global_configs` and `chain_configs`, so let's stick to that and rename this table to `fiat_payment_records`

Reply:
done. changed in the mongo db as well